packages/node_modules/@webex/internal-plugin-team/src/team.js

```diff
@@ -2,9 +2,10 @@
  * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
  */
 
+import querystring from 'querystring';
```

Comment:
Why move this?

Reply:
I did an `npm --fix` in this file. According to the eslint rule, this line needed to move

Reply:
We must bow to our robot overlords 🤖